client/apps/Admin/src/components/CreateProjectModal.tsx

```
@@ -1,28 +1,264 @@
+              />
+              {errors.description && (
+                <span className="user-form__field-error">{errors.description.message}</span>
+              )}
+            </div>
+
+            <div className="user-form__field">
+              <label className="user-form__label">
+                סוג דיווח <span className="user-form__required">*</span>
+              </label>
+              <div style={{ display: 'flex', gap: '1rem', flexDirection: 'row-reverse' }}>
+                <label style={{ display: 'flex', alignItems: 'center', gap: '0.5rem', cursor: 'pointer' }}>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)